src/ui/variables/charts.js

```diff
@@ -185,8 +185,7 @@ const completedTasksChart = {
   },
 };
 
-module.exports = {
-  dailySalesChart,
-  emailsSubscriptionChart,
-  completedTasksChart,
-};
+// module.exports = {
```

Comment:
If these are covered by the above `export const` can we remove the comments? No need to keep the code if it isn't being used.